src/sentry/api/serializers/rest_framework/rule.py

```diff
@@ -67,8 +67,11 @@ class RuleSerializer(serializers.Serializer):
     actionMatch = serializers.ChoiceField(
         choices=(("all", "all"), ("any", "any"), ("none", "none"))
     )
+    filterMatch = serializers.ChoiceField(
+        choices=(("all", "all"), ("any", "any"), ("none", "none")), required=False
+    )
     actions = ListField(child=RuleNodeField(type="action/event"))
-    conditions = ListField(child=RuleNodeField(type="condition/event"))
+    conditions = ListField(child=RuleNodeField(type=None))
```

Comment:
This is so that filters can be passed alongside conditions from the frontend. If this is too jank I can change type to accept an array of valid `RuleBase` types (e.g. `["condition/event", "filter/event"]`)

Reply:
>  I can change type to accept an array of valid `RuleBase` types (e.g. `["condition/event", "filter/event"]`)

I personally prefer this approach

Reply:
I ended up adding a separate ListField for filters so that validation would make more sense, previously filter validation errors would be grouped with conditions which made it hard to display properly in the frontend.

I then group the filters + conditions into one big conditions array after validation